stdlib/public/runtime/Errors.cpp

```diff
@@ -471,3 +471,32 @@ void swift::swift_abortDisabledUnicodeSupport() {
                     "Unicode normalization data is disabled on this platform");
 
 }
+
+#if defined(_WIN32)
+// On Windows, exceptions may be swallowed in some cases and the
+// process may not terminate as expected on crashes. For example,
+// illegal instructions used by llvm.trap. Disable the exception
+// swallowing so that the error handling works as expected.
```

Comment:
Perhaps we should also document why a last chance vectored exception handler would be insufficient for this.

Reply:
Haven't tried it yet. Will try. But this works as is.